### PR TITLE
Create table

### PR DESCRIPTION
**Pull request by karam084**
**Project requirements**

- Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
- Create a database named vet_clinic.
- Create a table named animals with the following columns:
- id: integer
- name: string
- date_of_birth: date
- escape_attempts: integer
- neutered: boolean
- weight_kg: decimal
- Copy the SQL you used in the previous point into the schema.sql file in the generated repository.
- Insert the following data:
- Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
- Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
- Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
- Copy the SQL you used in the previous point into the data.sql file in the generated repository.
- Write queries for the following quests:
- Find all animals whose name ends in "mon".
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.
- List the date of birth of all animals named either "Agumon" or "Pikachu".
- List name and escape attempts of animals that weigh more than 10.5kg
- Find all animals that are neutered.
- Find all animals not named Gabumon.
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)

**Here are the query results**
 - Find all animals whose name ends in "mon."
  Screenshot from 2022-09-15 03-52-26
 - List the name of all animals born between 2016 and 2019.
   Screenshot from 2022-09-15 03-53-06
- List the name of all animals that are neutered and have less than three escape attempts.
 
   Screenshot from 2022-09-15 03-53-45
 - List the date of birth of all animals named "Agumon" or "Pikachu."
 Screenshot from 2022-09-15 04-01-38
 - List names and escape attempts of animals that weigh more than 10.5kg.
 Screenshot from 2022-09-15 04-02-07
 - Find all animals that are neutered.
 Screenshot from 2022-09-15 04-02-27 
 - Find all animals not named Gabumon.
 Screenshot from 2022-09-15 04-02-47
 - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg).
   Screenshot from 2022-09-16 02-07-19

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon%';
+select * from animals where date_of_birth between '2016-01-01' and '2019-12-31';
+select * from animals where neutered = true and escape_attempts < 3;
+select date_of_birth from animals where name = 'Agumon' or name = 'Pikachu';
+select name, escape_attempts from animals where weight_kg > 10.5;
+select * from animals where neutered = true;
+select * from animals where name != 'Gabumon';
+select * from animals where weight_kg >= 10.4 and weight_kg <= 17.3;
```
Comment (BenMukebo) [inline queries.sql:10]: - [x] Excellent job writing queries for the following quests👏🏻 however it's one of the project requirements you take a screenshot of the results of your queries and add them to the pull request description. This also helps to ensure that you run the queries with success. Kindly make this 👍🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @karam084,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
 
- Good commit messages ✔️ 
- Well create the `animals` table. ✔️ 
- Well Insert data into the animal table.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (karam084) [reply queries.sql:10]: Thank you

Comment (BenMukebo) [reply queries.sql:10]: Hi @karam084 !! The screenshot of the results of your queries must be added to the pull request mean here!!

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @karam084,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
 
- Good commit messages ✔️ 
- Well create the `animals` table. ✔️ 
- Well Insert data into the animal table.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [x] Excellent job adding the creenshot of the results of your queries to the README 👏🏻 however as required by the previous reviewer,  it's one of the project requirements you add the results of your queries to your `pull request` description. This also helps to ensure that you run the queries with success. Kindly make this 👍🏻 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by karam084** [comment]
Thank you

**Review by Lordkaito — APPROVED** [review]
Hi @karam084, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by karam084** [comment]
Thank you

**Pull request merged**